samples/oauth_rails/VetsApiAppeals/app/views/layouts/application.html.erb

```diff
@@ -16,6 +16,7 @@
   <body>
     <header class="site-header" role="banner">
       <div class="site-l-wrapper">
+        <div class="vads-u-text-align--right vads-u-padding-top--2p5"><%= Figaro.env.vets_api_url %></div>
```

Comment:
Looks good! For curiosity's sake, should this display the whole `api_url`, or just the env derived from the url?

Reply:
I think the whole URL is fine. No need to get terribly fancy with it.

Reply:
^ I wondered the same thing...but I think it will ultimately be up to the user to set this up (top tip from Drew).